scripts/install-build-deps

```diff
@@ -136,14 +136,83 @@ zfs
 END_TAGS
 )
 
-# python3 packages available starting from CentOS/RedHat 7.7
-if (( $(grep -Po '(?<=release 7\.).' /etc/redhat-release) < 7 )) ; then
-    skip_tags+=,motr-build-python3
-fi
-
 if ! $install_epel ; then
     skip_tags+=,epel
 fi
 
+packages_redhat=(
+git
+libtool
+autoconf
+make
+rpm-build
+gcc
+gcc-c++
+clang
+clang-libs
```

Comment:
We do have this list of packages in the cortx-motr.spec file already, so why don't we use `yum-builddep` to download and install them? This way we will avoid the need to maintain this list in two places.

Reply:
Good idea, updated the patch with it.
There is one issue if kernel or lustre client packages are not available, but which can be ignored.